frontend/src/lib/newQueryBuilder/queryBuilderMappers/mapCompositeQueryFromQuery.ts

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable  */
+// @ts-ignore
+// @ts-nocheck
+
+import { PANEL_TYPES } from 'constants/queryBuilder';
+import { ICompositeMetricQuery } from 'types/api/alerts/compositeQuery';
+import {
+	IClickHouseQuery,
+	Query,
+} from 'types/api/queryBuilder/queryBuilderData';
+import { EQueryType } from 'types/common/dashboard';
+
+import { mapQueryDataToApi } from './mapQueryDataToApi';
+
+export const mapCompositeQueryFromQuery = (
+	query: Query,
+	panelType: PANEL_TYPES | null,
+): ICompositeMetricQuery => {
+	const queryData = query[query.queryType];
+	const QueryPayload: {
+		compositeQuery: ICompositeMetricQuery;
+	} = {
+		compositeQuery: {
+			queryType: query.queryType,
+			panelType: panelType || PANEL_TYPES.TIME_SERIES,
+			builderQueries: {},
+			chQueries: {},
+			promQueries: {},
+		},
+	};
+
+	switch (query.queryType) {
+		case EQueryType.QUERY_BUILDER: {
```

Comment:
this need to handled via mapper function

Reply:
Done.